### ISSUE TITLE
porter param generate is a no-op for local bundle

### ISSUE THREAD

**Issue by squillace** [getporter/porter#2784]
## Describe the bug
`porter param generate` should generate a customized parameter set based on the current local bundle. It does not in porter porter v1.0.14 (0e739d88).

## To Reproduce
1. Run `porter create` to create a new hello-porter bundle.
2. Modify the porter.yaml to uncomment the mysql parameter
3. Run this porter command `porter param generate`
4. Run `ls` and note that there is no new param set file.
5. Run `porter param list` and note that there is no new param set.

## Expected behavior
A param set should be created.

## Porter Command and Output
```
squillace@idiopath:~/work/squillace/howdy$ porter param generate myfoo
Generating new parameter set myfoo from bundle porter-hello
==> 1 parameter(s) declared for bundle porter-hello
? How would you like to set parameter "mysql_user"
  environment variable
? Enter the environment variable that will be used to set parameter "mysql_user"
  HELLO
squillace@idiopath:~/work/squillace/howdy$ ls
README.md  helpers.sh  porter.yaml  template.Dockerfile
```

## Version
Copy the output of `porter version` below

porter v1.0.14 (0e739d88)

**Comment by squillace** [comment]
update: this is not quite true. But it DOES create a param that is quietly -- very quietly -- stored in the state db (mongo). As a result, however, you cannot see it unless you remember the name.

**Comment by squillace** [comment]
it's possible that really the only problem here is that `porter params list` doesn't work, specified in #2785 
```
squillace@idiopath:~/work/squillace/howdy$ porter show porter-hello
Name: porter-hello
Namespace:
Created: 6 minutes ago
Modified: 18 seconds ago

Parameters:
--------------------------------------------
  Name        Type    Value
--------------------------------------------
  mysql_user  string  not different at all
  ralph       string  not different at all

Parameter Sets:
  - ralph

Status:
  Reference:
  Version: 0.1.0
  Last Action: upgrade
  Status: succeeded
  Digest:
squillace@idiopath:~/work/squillace/howdy$ porter list
--------------------------------------------------------------------------
  NAMESPACE  NAME          VERSION  STATE      STATUS     MODIFIED
--------------------------------------------------------------------------
             porter-hello  0.1.0    installed  succeeded  35 seconds ago
squillace@idiopath:~/work/squillace/howdy$ porter params list
-----------------------------
  NAMESPACE  NAME  MODIFIED
-----------------------------
```

**Comment by AGMETEOR** [comment]
Hey @troy0820 your PR https://github.com/getporter/porter/pull/2824 should also be able to close this?

**Comment by troy0820** [comment]
Yes @AGMETEOR @squillace I believe my work in #2824 will close this issue as well.

**Comment by troy0820** [comment]
![image](https://github.com/getporter/porter/assets/5985850/13512916-f293-49d5-893b-0a359f749b4e)

@squillace this has been fixed with the recent addition to the PR that has been merged #2824
